src/components/FieldValueDirectAnswer.tsx

```diff
@@ -0,0 +1,124 @@
+import {
+  FieldValueDirectAnswer as FieldValueDirectAnswerType,
+  BuiltInFieldType,
+  Address
+} from '@yext/search-headless-react';
+import { useMemo } from 'react';
+
+/**
+ * Props for FieldValueDirectAnswer.
+ */
+interface FieldValueDirectAnswerProps {
+  /** A field value direct answer result. */
+  result: FieldValueDirectAnswerType,
+  /** Handle onClick event for "View Details" link. */
+  viewDetailsClickHandler?: () => void,
+  /** CSS classes for customizing the component styling. */
+  cssClasses?: FieldValueDirectAnswerCssClasses
+}
+
+/**
+ *  The CSS class interface for FieldValueDirectAnswer.
+ */
+interface FieldValueDirectAnswerCssClasses {
+  header?: string,
+  body?: string,
+  content?: string,
+  answerContainer?: string
+}
+
+/**
+ * Renders a field value direct answer provided by the Search API.
+ *
+ * @internal
+ */
+export function FieldValueDirectAnswer({
+  result,
+  viewDetailsClickHandler,
+  cssClasses = {}
+}: FieldValueDirectAnswerProps): JSX.Element {
+  const title = `${result.entityName} / ${result.fieldName}`;
+  const link = result.relatedResult.link ?? result.relatedResult.rawData.landingPageUrl as string;
```

Comment:
this `link` logic will be the same regardless of DA type. what do you think about leaving it in `DirectAnswer` and passing it as a prop?

Reply:
I'm incline to keep link in the individual BlahDirectAnswer component, since it's based on what they would display